Production_scripts/B05b_Link_CUP_marker.R

```diff
@@ -0,0 +1,145 @@
+#####################################################
+# A&E Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description - Process A & E extract
+#####################################################
+
+# Load packages
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+library(createslf)
+library(slfhelper)
+library(readr)
+
+# Read in data---------------------------------------
+
+# Specify year
+year <- 1920
+
+## get data ##
+ae_cup_extract <- readr::read_csv(
```

Comment:
```suggestion
ae_cup_file<- readr::read_csv(
```
Ive renamed this for reading the raw data